Package.swift

```diff
@@ -90,37 +94,37 @@ let package = Package(
                 "_CoreFoundation"
             ],
             path: "Sources/Foundation",
-            swiftSettings:  [.define("DEPLOYMENT_RUNTIME_SWIFT"), .define("SWIFT_CORELIBS_FOUNDATION_HAS_THREADS")]
+            swiftSettings: swiftBuildSettings
         ),
         .target(
             name: "FoundationXML",
             dependencies: [
                 .product(name: "FoundationEssentials", package: "swift-foundation"),
-                "Foundation",
+                .targetItem(name: "Foundation", condition: nil),
```

Comment:
For my context, is there a difference between these two spellings?

Reply:
Oh - I missed this, sorry. It was required when I used a `String` instead of a string literal for the target name. We can revert it back now.